index.js

```diff
@@ -1362,7 +1362,29 @@
             };
           };
         };
-        return S.encase3(f, x, y, z);
+        return encase3(f, x, y, z);
+      });
+
+  //# maybeToEither :: a -> Maybe b -> Either a b
+  //.
+  //. Takes a value of any type and a Maybe, and returns an Either.
+  //. If the second argument is a Nothing, a Left containing the first
```

Comment:
Another possibility:

_Converts a Maybe to an Either. A Nothing becomes a Left (containing the first argument); a Just becomes a Right._

What do you think?